.github/workflows/label.yml

```diff
@@ -1,17 +0,0 @@
-on: pull_request_target
-
-jobs:
-  apply-label:
```

Comment:
This helps us easily identify incoming PRs. It is `pull_request_target` but we dont checkout code. Any reason why we are removing this?